### PR TITLE
ENG-3777: Run Python notebooks in SDK Circle-CI

### PR DESCRIPTION
**Pull request by JMkrish**
1. Added Python notebooks inside 'examples' dir to SDK unit test suite.
2. Made required changes to run the test cases in parallel using pytest-xdist.
3. Added the required environment variables to the context 'org-sightmachine' in CircleCI's project settings.

https://sightmachine.atlassian.net/browse/ENG-3777

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -51,18 +47,22 @@ jobs:
...
           pkg-manager: pip
       - run:
           name: Run Unit Tests
           command: |
             set -x
+            export ENV_VAR_TENANT=${ENV_VAR_TENANT}
+            export ENV_VAR_USERNAME=${ENV_VAR_USERNAME}
+            export ENV_VAR_PASSWORD=${ENV_VAR_PASSWORD}
+            export ENV_VAR_API_KEY=${ENV_VAR_API_KEY}
+            export ENV_VAR_API_SECRET=${ENV_VAR_API_SECRET}
```
Comment (mklein0) [inline .circleci/config.yml:61]: Are these not already exported environment variables?  Echoing sensitive data such as password/secrets is bad.  If needed, better to move the `set -x` later.

Comment (JMkrish) [reply .circleci/config.yml]: "You are correct. As I've already added these new environment variables to the project's context, there is no need to export them here. However, CircleCI is smart enough not to print this sensitive information, even if we try to display it.

I have removed these export statements and updated the 'config.yml'."

File: requirements-codecoverage.txt [hunk 2]
```diff
@@ -1,4 +1,8 @@
 # Use public repo for coverall upload
```
Comment (mklein0) [inline requirements-codecoverage.txt:1]: This requirements file really includes test libraries and code coverage libraries.

**Review by mklein0 — APPROVED** [review]
Looks good. Just a comment on naming/labeling.

File: requirements-codecoverage.txt (hunk 2 above)
Comment (JMkrish) [reply requirements-codecoverage.txt:1]: Moved 'ipywidgets' & 'plotly' libraries from requirements-codecoverage.txt to requirements.txt

**Pull request merged**